### ISSUE TITLE
Revise the Program Area: Access the data

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#2274]
### Overview
We need fix a misspelling of the program area information in the Access the Data project so that it will show up when we add the program areas filter.

### Action Items
- [x] Revise the text in the file indicated in the Resources section
- from
```
program area:  'Citizen Engament'
```
  - to
```
program-area:  
    - Citizen Engagement
```

### Resources/Instructions
File location to make the change: _projects/access-the-data.md

URL for UAT to check the page has been changed: https://github.com/hackforla/website/blob/gh-pages/_projects/access-the-data.md

**Comment by macho-catt** [comment]
Looks like #2259 is addressing this. We can close when that is merged.

**Comment by macho-catt** [comment]
re-addressed and fixed in #2333
